tests/lib/rules/no-unsupported-features/es-syntax.js

```diff
@@ -876,15 +1036,19 @@ runTests([
                 options: [{ version: "10.0.0" }],
                 errors: [
                     {
-                        messageId: "no-modules",
-                        data: { supported: null, version: "10.0.0" },
+                        messageId: "not-supported-till",
+                        data: {
+                            ruleId: "no-modules",
+                            supported: "^12.17.0 || >=13.2.0",
+                            version: "10.0.0",
+                        },
                     },
                 ],
             },
         ],
     },
     {
-        keyword: "new.target",
+        keyword: "newTarget",
```

Comment:
There are a couple of these that I changed, this is in order to match the rule name in `eslint-plugin-es-x`. I was thinking of adding an alias mechanism here, but I think that may get out of hand a little.
| Before | After |
| --- | --- |
| `new.target` | `newTarget` |
| `regexpLookbehind` | `regexpLookbehindAssertions` |
| `regexpS` | `regexpSFlag` |
| `regexpU` | `regexpUFlag` |
| `regexpUnicodeProperties` | `regexpUnicodePropertyEscapes` |
| `regexpY` | `regexpYFlag` |
| `trailingCommasInFunctions` | `trailingFunctionCommas` |
| `unicodeCodePointEscapes` | `unicodeCodepointEscapes` |

Reply:
I like to be having the same name with the plugin, but, for compat, we may need to a layer to normalize the current name to the new one.